Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -49,10 +49,12 @@ public bool IsValid(EntityWhitelist list, EntityUid uid)
     {
         if (list.Components != null)
```

Comment:
It was brought to my attention by @stalengd that my change breaks API.
He's proposing the following solution:
```suggestion
        if (list.Components != null)
        {
            var regs = StringsToRegs(list.Components);

            list.Registrations ??= new List<ComponentRegistration>();
            list.Registrations.Clear();
            list.Registrations.AddRange(regs);
        }
```